File: server/src/models/ProductModel.js
const mongoose = require("mongoose");

const schema = mongoose.Schema

const productSchema = new schema({
    product_name: { type: String },
    quantity: { type: String },
    price: { type: String },
    description: { type: String },
    category: { type: String },
    product_image: { type: String },
    total:{type:String},
    total_amount:{type:String},
    
})

const productModel = mongoose.model('product_tb',productSchema)
module.exports = productModel

File: lib/Admin/model/productmodel.dart
class productModel {
  final String productname;
  final String category;
  final String quantity;
  final String price;
  final String description;
  final String id;
  final String image;


  productModel({required this.productname,
    required this.category,
    required this.quantity,
    required this.price,
    required this.description,
    required this.image,

    required this.id,

  });

  factory productModel.fromJson(Map<String, dynamic> json) {
    return productModel(
      productname: json['product_name'],
      category: json['category'],
      quantity: json['quantity'],
      price: json['price'],
      description: json['description'],
      image: json['product_image'],
      id: json['_id'],

    );
  }
}